.eslintrc.js

```diff
@@ -5,9 +5,5 @@ module.exports = {
     ],
     'plugins': [
         'react'
-    ],
-    'rules': {
```

Comment:
These rules are handled by `eslint-config-jitsi`.
We no longer need to overrride them as warnings. We have already disabled them in this commit - 6f35ddd792c5a3e9cc462c31cb1166fa1fbbdb09